src/main/java/org/datadog/jenkins/plugins/datadog/model/BuildPipeline.java

```diff
@@ -133,6 +133,7 @@ private void completeInformation(final List<BuildPipelineNode> nodes, final Buil
                 final BuildPipelineNode executableChildNode = searchExecutableChildNode(node);
                 if(executableChildNode != null) {
                     node.setPropagatedNodeName(executableChildNode.getNodeName());
+                    node.setPropagatedNodeLabels(executableChildNode.getNodeLabels());
```

Comment:
are you propagating it from job spans to stage/pipeline? I guess that's accurate for Jenkins?

Reply:
Yes, the worker information only is available in an actual executable child node, which is a `job`.